src/dmd/target.d

```diff
@@ -487,6 +489,25 @@ struct Target
         return null;
     }
 
+    /**
+     * Returns the type which will really be used for passing the given
+     * parameter to an extern(C++) function.
+     */
+    extern (C++) static Type cppParameterType(Parameter p)
```

Comment:
Should have a `Params` element n the DDoc header.

Reply:
"really" sounds strange.  Did you mean "ultimately" or "eventually"?

Reply:
@JinShil - I'll ddoc the rest of target for consistency in another PR, as this module is my baby. ;-)

Reply:
Using "really" makes sense to me.  For example, a `lazy bool` will "really" be passed as a `bool delegate()`